test/unit/user_test.rb

```diff
@@ -119,4 +119,27 @@ def test_create_user_successful_service_not_require_registered_users
 
     assert_equal metrics, user.load_metric_names
   end
+
+  test '.delete_all removes all the Users of a Service' do
```

Comment:
I think we should also test the case of deleting the users of a service that does not have any. Just to make sure that nothing crashes.

Reply:
:+1: